.github/workflows/.trivyignore

```diff
@@ -1,7 +1,3 @@
 # Jul 16 2025
 # Issue with commons-fileupload, needs an update of spring-cloud-openfeign
 CVE-2025-48976
```

Comment:
Is this still relevant?

Reply:
Yes checked and this is still an relevant (they are working on a new major version)